packages/core-components/src/layout/SignInPage/commonProvider.tsx

```diff
@@ -33,13 +33,18 @@ const Component: ProviderComponent = ({ config, onSignInSuccess }) => {
   const { apiRef, title, message } = config as SignInProviderConfig;
   const authApi = useApi(apiRef);
   const errorApi = useApi(errorApiRef);
+  const configApi = useApi(configApiRef);
+  const usePopup = configApi.getOptionalBoolean('auth.usePopup') ?? true;
 
   const handleLogin = async () => {
     try {
       const identityResponse = await authApi.getBackstageIdentity({
-        instantPopup: true,
+        instantPopup: usePopup,
```

Comment:
This should likely stay true regardless, as it just means it bypasses the modal. The `DefaultAuthConnector` will need to handle this by instantly redirecting rather than opening a popup in `.createSession`